packages/cdktf-cli/src/bin/cmds/__tests__/handlers-login.test.ts

```diff
@@ -0,0 +1,82 @@
+import { login } from "../handlers";
```

Comment:
I wanted to add a regression test for this issue, but as you can see it's a lot of work, and is a bit 'icky', to use a professional term. Ideally, if there was more time, I'd refactor this a bit more, but I doubt we can do away with mocking a _lot_ of things. 

What do you think? We should try to aim for having more of these kinds of tests for this level of our CLI, since we don't seem to have many any right now.

Reply:
I'm unsure, I sort of like them, but these are also the kind of tests that might break easily when we do internal refactorings. Maybe a CLI e2e test would be an alternative 🤔